app/presenters/reservation_user_action_presenter.rb

```diff
@@ -131,7 +133,12 @@ def move_link
   end
 
   def report_an_issue_link
-    link_to(I18n.t("views.instrument_issues.new.title"), new_facility_order_order_detail_issue_path(facility, order, order_detail))
+    if @redirect_to_order_id.present?
```

Comment:
Consider wrapping this in a method for documentation and clarity like: `modal_display?`